autoload/neomake/utils.vim

```diff
@@ -297,3 +297,8 @@ function! neomake#utils#redir(cmd) abort
     endtry
     return neomake_redir
 endfunction
+
+function! neomake#utils#ExpandArgs(args) abort
+    " Only expand those args that start with \ and a single %
+    call map(a:args, "v:val =~# '\\(^\\\\\\|^%$\\|^%[^%]\\)' ? expand(v:val) : v:val")
```

Comment:
I don't understand how this regex works. maybe this needs to be a double quote string?